chia/rpc/wallet_rpc_api.py

```diff
@@ -1077,7 +1077,7 @@ async def rl_set_user_info(self, request):
             await rl_user.set_user_info(
                 uint64(request["interval"]),
                 uint64(request["limit"]),
-                origin["parent_coin_info"],
+                origin["parent_coin_id"],
```

Comment:
this looks like it changes the RPC API.. is it? and are we OK with that? Perhaps it would be safer to include this value under both the old and new field name

Reply:
I guess nobody is using this API anyway..

Reply:
True, this will break a feature in the Rate Limited Wallet. I'll take a few minutes to consider fixing that, or also returning `parent_coin_info`

Reply:
Surface area looks small in the GUI - I'll submit a linked PR for the GUI, too.

Reply:
https://github.com/Chia-Network/chia-blockchain-gui/pull/495